ConMan.java

```diff
@@ -1,64 +1,47 @@
+import java.io.InputStream;
 import java.io.PrintWriter;
 import java.io.Writer;
 import java.util.ArrayList;
 
 
 public class ConMan {
     private final UI ui;
+    private final InputStream stream;
     private ConsoleMenu menu;
-    public ArrayList<Contact> allContacts = new ArrayList<>();
+    private ArrayList<Contact> allContacts = new ArrayList<>();
 
-    public ConMan(ConsoleMenu consoleMenu, UI ui) {
+    public ConMan(ConsoleMenu consoleMenu, UI ui, InputStream stream) {
         this.menu = consoleMenu;
         this.ui = ui;
-        //allContacts.add(new Contact("Will", "Curry", "Tele", "Email"));
+        this.stream = stream;
```

Comment:
You've inject an InputStream here, and it's only use is in the constructor of ConsoleMenu, however you've already passed in a ConsoleMenu object in this constructor.